File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/cmd/entity/GetVmsTerminalSensorParamsEntity.java
package com.shmedo.configlibrary.iot.cmd.entity;

import com.shmedo.configlibrary.ble.interfaces.Validater;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  11/23/20 <br/>
 * 描述：     获取Vms终端某个通道下传感器参数
 */
public class GetVmsTerminalSensorParamsEntity implements Validater {
    private String sn;
    private int channel;

    public GetVmsTerminalSensorParamsEntity(String sn, int channel) {
        this.sn = sn;
        this.channel = channel;
    }

    @Override
    public void validate() {

    }

    @Override
    public String toString() {
        return "sn=" + sn + "&channel=" + channel;
    }
}

File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/cmd/entity/vms/GetVmsTerminalSensorParamsEntity.kt
package com.shmedo.configlibrary.iot.cmd.entity.vms

import com.shmedo.configlibrary.ble.interfaces.Validater

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  11/23/20 <br></br>
 * 描述：     获取Vms终端某个通道下传感器参数
 */
class GetVmsTerminalSensorParamsEntity(private val sn: String, private val channel: Int) :
    Validater {
    override fun validate() {}
    override fun toString(): String {
        return "sn=$sn&channel=$channel"
    }
}